### PR TITLE
Add expansion menu

### PR DESCRIPTION
**Pull request by paulinembabu**
See issue https://github.com/OpenSRP/opensrp-client-emmunize-malawi/issues/17

### CODE REVIEW

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/dao/ReportDao.java [hunk 1]
```diff
@@ -269,8 +269,15 @@ private static void initVaccine(String vaccineCategory,
                 }
                 List<Alert> alerts = new ArrayList<>();
                 for (Alert alert : raw_alerts) {
-                    if (alert.startDate() != null && alert.status() != AlertStatus.complete && !myGivenVaccines.contains(cleanName(alert.visitCode())))
-                        alerts.add(alert);
+                    if (alert.startDate() != null && alert.status() != AlertStatus.complete && !myGivenVaccines.contains(cleanName(alert.visitCode()))){
+                        if(!alert.scheduleName().equalsIgnoreCase("OPV 0")){
```
Comment (bennsimon) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/dao/ReportDao.java:273]: swap this

File: opensrp-giz-malawi/src/test/java/org/smartregister/giz/activity/ReportSummaryActivityTest.java [hunk 2]
```diff
@@ -48,7 +51,7 @@ public void setUp() {
     }
 
     @Test
-    public void testOnResume() {
+    public void testOnResumeSetsViewVisible() {
```
Comment (bennsimon) [inline opensrp-giz-malawi/src/test/java/org/smartregister/giz/activity/ReportSummaryActivityTest.java:54]: remove this and add `resume()` on line 49

**Review by bennsimon — CHANGES_REQUESTED** [review]

File: opensrp-giz-malawi/src/test/java/org/smartregister/giz/activity/ReportSummaryActivityTest.java (hunk 2 above)
Comment (paulinembabu) [reply opensrp-giz-malawi/src/test/java/org/smartregister/giz/activity/ReportSummaryActivityTest.java:54]: When I move this then it skips the part where it's supposed to actually check on the Visibility

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/dao/ReportDao.java (hunk 1 above)
Comment (bennsimon) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/dao/ReportDao.java:273]:
```suggestion
                        if(!"OPV 0".equalsIgnoreCase(alert.scheduleName())){
```

**Review by bennsimon — CHANGES_REQUESTED** [review]

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/dao/ReportDao.java (hunk 1 above)
Comment (paulinembabu) [reply opensrp-giz-malawi/src/main/java/org/smartregister/giz/dao/ReportDao.java:273]: Done

Comment (paulinembabu) [reply opensrp-giz-malawi/src/main/java/org/smartregister/giz/dao/ReportDao.java:273]: Done

File: opensrp-giz-malawi/src/test/java/org/smartregister/giz/activity/ReportSummaryActivityTest.java (hunk 2 above)
Comment (paulinembabu) [reply opensrp-giz-malawi/src/test/java/org/smartregister/giz/activity/ReportSummaryActivityTest.java]: Done

**Review by bennsimon — APPROVED** [review]

**Pull request merged**